libraries/Microsoft.Bot.Builder/IBotFrameworkStreamingChannelConnector.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+using Microsoft.Rest;
+
+namespace Microsoft.Bot.Builder
+{
+    /// <summary>
+    /// This is a collection of methods that must be implemented on adapters supporting channels built on 
+    /// the Streaming Extensions feature set of the V3 protocol. They are equivalent to the HTTP endpoints 
+    /// defined in Bot.Connector but are modified to be communicated over a persistent, streaming, connection.
+    /// </summary>
+    public interface IBotFrameworkStreamingChannelConnector
```

Comment:
I thought we had planned to have different return types, I'll look at what ASE currently does and update this if needed, or start the conversation on nuking it otherwise. The original intent was to have an interface that acts as the source of truth for conversation calls to "streaming" channels, since the channel sdk they're using doesn't strictly enforce the current interface and we wanted to make sure the channels and bots were in sync. I have a feeling the return types of these methods are out of date, I'll find out for sure and have it addressed soon.